### PR TITLE
Use a Galaxy root path with a space for integration tests

### PR DESCRIPTION
**Pull request by nsoranzo**
and fix uncovered bugs.

### CODE REVIEW

File: lib/galaxy/tool_util/deps/commands.py [hunk 1]
```diff
@@ -118,16 +118,12 @@ def _wait(cmds, **popen_kwds):
     return stdout
 
 
-def download_command(url, to=STDOUT_INDICATOR, quote_url=False):
+def download_command(url, to=STDOUT_INDICATOR):
```
Comment (jmchilton) [inline lib/galaxy/tool_util/deps/commands.py:121]: quote_url is still used in Planemo right? Why break backward compatibility on this?

Comment (nsoranzo) [reply lib/galaxy/tool_util/deps/commands.py:121]: I forgot about that! It's quite unsecure though; if I fix it in Planemo, would it be OK? Or would you like to keep it any way?

**Comment by nsoranzo** [comment]
Green and ready for review!

File: lib/galaxy/tool_util/deps/commands.py (hunk 1 above)
Comment (nsoranzo) [reply lib/galaxy/tool_util/deps/commands.py:121]: Done in https://github.com/galaxyproject/planemo/pull/984

**Review by mvdbeek — APPROVED** [review]

**Pull request merged**